framework/PageDashboard/PageDashboardChart.tsx

```diff
@@ -83,9 +85,20 @@ export function PageDashboardChart(props: {
           }
           containerComponent={
             <ChartVoronoiContainer
-              labels={(point: { datum: { x: string | number; y: string | number } }) => {
+              labels={(point: {
+                datum: {
+                  x: string | number;
+                  y: string | number;
+                  hosts_added?: string | number;
+                  hosts_deleted?: string | number;
+                };
+              }) => {
                 const datum = point.datum;
-                return `${datum.x}: ${datum.y}`;
+                return yLabel == 'Unique hosts' && datum.hosts_added !== undefined
```

Comment:
I thin this should be translated.

Reply:
These chart usage specific strings should not be in the generic PageDashboardChart.